main/main.cpp

```diff
@@ -722,6 +722,28 @@ Error Main::setup(const char *execpath, int argc, char *argv[], bool p_second_ph
 		I = N;
 	}
 
+	FileAccessNetwork::configure();
```

Comment:
Might be worth adding some comments to make clear why it has to happen here and not after `globals->setup`.

Reply:
Maybe it can be done after `globals->setup()`. I'll give a check.

Reply:
Well I didn't mean that it has to move, but yeah that proves my point that a comment explaining why it's here and not someplace else could be good :) (since you had to move it to fix a bug).

Reply:
I meant just the `configure()` call. The block itself has to be moved, because `globals->setup()` will look for `project.godot`, which will only be found after the network FS is enabled.